crates/router/src/core/routing/helpers.rs

```diff
@@ -283,6 +283,7 @@ pub async fn update_business_profile_active_algorithm_ref(
         extended_card_info_config: None,
         use_billing_as_payment_method_billing: None,
         collect_shipping_details_from_wallet_connector: None,
+        collect_billing_details_from_wallet_connector: None,
```

Comment:
can you create a separate enum variant for this? maybe something like routing update, this enum variant would have zero fields and internally updates the modified time only